mobile/lib/modules/asset_viewer/ui/video_player.dart

```diff
@@ -21,6 +22,7 @@ class VideoPlayerViewer extends HookConsumerWidget {
     required this.hideControlsTimer,
     required this.showControls,
     required this.showDownloadingIndicator,
+    this.autoPlayVideo = true,
```

Comment:
Can we change this to `autoPlay` to be consistent with the params to the chewie controller